osu.Game.Rulesets.Osu/Objects/Slider.cs

```diff
@@ -155,13 +155,20 @@ private void createRepeatPoints()
             var repeatPointDistance = Math.Min(Distance, length);
             var repeatDuration = length / Velocity;
 
+            bool sliderStart = true;
+
             for (var repeat = 1; repeat < RepeatCount; repeat++)
             {
+                sliderStart = !sliderStart;
+
                 for (var d = repeatPointDistance; d <= length; d += repeatPointDistance)
                 {
                     var repeatStartTime = StartTime + repeat * repeatDuration;
                     var distanceProgress = d / length;
 
+                    if (sliderStart)
+                        distanceProgress = 0;

```

Comment:
why is this needed?

Reply:
this makes even numbered repeats where the start circle is like normal sliders, before they were always at the end circle